src/jobs/test.yml

```diff
@@ -0,0 +1,33 @@
+description: |
+  Simple drop-in job to test your PHP application automatically.
+
+parameters:
+  version:
+    type: string
+    description: The `cimg/php` Docker image version tag.
+  setup:
+    type: steps
+    description: Provide any optional steps you would like to run prior to installing the node dependencies. This is a good place to install global packages.
```

Comment:
Small typo here (I think?) s/node/php

Reply:
done in https://github.com/TangRufus/php-orb/commit/8135e650af645f923997aee740a39869dedc99c8